pandas/formats/format.py

```diff
@@ -910,15 +930,94 @@ def get_col_type(dtype):
                          if x else '{}') for x in row]
             else:
                 crow = [x if x else '{}' for x in row]
+            if i < clevels and self.fmt.header and self.multicolumn:
+                # sum up columns to multicolumns
+                crow = self._format_multicolumn(crow, ilevels)
+            if i >= nlevels and self.fmt.index and self.multirow and\
+                    ilevels > 1:
```

Comment:
small style issue: in pandas we try to not use `\`, but rather use `(...)` for line continuation